shared/src/business/entities/cases/EligibleCase.ts

```diff
@@ -16,6 +17,7 @@ export class EligibleCase extends JoiValidationEntity {
   public irsPractitioners?: IrsPractitioner[];
   public privatePractitioners?: PrivatePractitioner[];
   public qcCompleteForTrial?: Record<string, any>;
+  public isSealed: boolean;
```

Comment:
We are showing the Sealed icons for internal users on trial session case lists as well.